src/ILLink.RoslynAnalyzer/TrimAnalysis/ArrayValue.cs

```diff
@@ -9,7 +9,12 @@ namespace ILLink.Shared.TrimAnalysis
 {
 	partial record ArrayValue
 	{
-		public ArrayValue (SingleValue size) => Size = size;
+		public SingleValue ElementType;
```

Comment:
Can you please revert this file? We don't really know what we're going to need for arrays yet.

Reply:
Oops, completely forgot this, thanks!